sig-robotics/proposal/Generating Action Sequences for Robot Task Plans using LLM/README.md

```diff
@@ -0,0 +1,167 @@
+<center>
+    <h1>Generating Action Sequences for Robot Task Plans using LLM</h1>
+</center>
+
+
+# 1. Background
+Since Google proposed the SayCan framework, empowering robots with large language models to interpret complex task instructions has become a popular research topic. The demand for robots is no longer limited to simple command execution; instead, there is a desire for semantic understanding and task execution based on the exploration of underlying needs. Technologies like ChatGPT have driven the advancement of this field, while RoboSDK and ROS, which provide a unified API for cloud-based, robot-centric applications, can fulfill data collection and task execution requirements for heterogeneous devices.
+
+This project use **prompt technique** to drive complex Robot operations, user utilize a prompt like  `Go to the kitchen and help me get an apple.` This prompt makes Robot go to the kitchen and find the apple, take it away, and finally take it back to the owner.
+
+Current research：
+- google [SayCan](https://say-can.github.io/)
+- Microsoft [PromptCraft-Robotics](https://github.com/microsoft/PromptCraft-Robotics/tree/main)
+- Nvidia [VOYAGER](https://github.com/MineDojo/Voyager)
+
+This proposal provides a basic robot LLM Agent (RoboAgent) for RoboSDK, enabling robot developers and users to use LLM prompt to drive robots in an intelligent manner.
+
+# 2. Goals
+For developers or end users of RoboSDK, the goals of the RoboAgent are:
+
+1. Test the perception, planning, and control algorithms of a robot in complex navigation and grasping tasks, both indoors and outdoors, based on given instructions (e.g. "Go to the front of that BYD car and check if anything important has been left behind"). Utilize large language models like ChatGPT for this purpose.
```

Comment:
1. emmm... Just a suggestion, you can simply use the above example rather than suppose a new one.
2. I assume you're referring to openai's `gpt-3.5-turbo` here, not `ChatGPT`.
3. Similarly, it might be a good idea to add reference links to external technologies.